diff --git a/node_modules/react-full-page/src/components/Controls.jsx b/node_modules/react-full-page/src/components/Controls.jsx
index b398caf..ee3077f 100644
--- a/node_modules/react-full-page/src/components/Controls.jsx
+++ b/node_modules/react-full-page/src/components/Controls.jsx
@@ -29,7 +29,7 @@ export default class Controls extends React.Component {
           disabled={currentSlideIndex === 0}
           onClick={this.props.onPrev}
         >
-          ←
+          왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽왼쪽
         </button>
         {this.renderSlidesNumbers(currentSlideIndex)}
         <button
